skfp/fingerprints/atom_pair.py

```diff
@@ -63,6 +64,11 @@ class AtomPairFingerprint(FingerprintTransformer):
     count : bool, default=False
         Whether to return binary (bit) features, or their counts.
 
+    normalize: bool, default=False
```

Comment:
I would change this to `scale_by_hac`, to be more precize. Molecule normalization also exists, and is something else